.github/workflows/linux_conditional_compilation.yml

```diff
@@ -72,17 +100,9 @@ jobs:
       SELECTIVE_BUILD_STAT_DIR: /__w/openvino/openvino/selective_build_stat
       MODELS_PATH: /__w/openvino/openvino/testdata
       SCCACHE_AZURE_KEY_PREFIX: ubuntu22_x86_64_itt_clang_Release
-    if: ${{ !needs.smart_ci.outputs.skip_workflow && github.event_name != 'merge_group' }}
+    if: ${{ !needs.Smart_CI.outputs.skip_workflow && github.event_name != 'merge_group' }}
```

Comment:
Is it just for consistency? We usually kept it in lower case

Reply:
autoformatter did the thing, reverted.